stormpath/src/main/java/com/stormpath/sdk/AndroidLogger.java

```diff
@@ -0,0 +1,35 @@
+package com.stormpath.sdk;
+
+import android.util.Log;
+
+public class AndroidLogger extends StormpathLogger {
+
+    public static final String TAG = "Stormpath";
+
+    @Override
+    protected void log(@LogLevel int logLevel, String message) {
+        switch (logLevel) {
```

Comment:
The log levels are now spelled out for the public method `Stormpath.setLogLevel()` and Android Studio will also complain if we try to pass an invalid value. I don't think we also need to spell it out on all internal methods. What do you think?